pkg/controller/controller_test.go

```diff
@@ -2513,15 +2529,21 @@ func assertServiceBindingOperationSuccessWithParameters(t *testing.T, obj runtim
 	case v1beta1.ServiceBindingOperationUnbind:
 		assertServiceBindingExternalPropertiesNil(t, obj)
 	}
+	assertServiceBindingUnbindStatus(t, obj, unbindStatus)
```

Comment:
We need to add assert for the `UnbindStatus` to the other state asserts for ServiceBinding. The goal is for every test to verify that `UnbindStatus` is correct every time that the status of a ServiceBinding is updated.

```
assertServiceBindingErrorBeforeRequest
assertServiceBindingFailedBeforeRequest
assertServiceBindingOperationInProgressWithParameters
assertServiceBindingStartingOrphanMitigation
assertServiceBindingErrorFetchingBinding
assertServiceBindingErrorInjectingCredentials
assertServiceBindingRequestRetriableErrorWithParameters
assertServiceBindingAsyncInProgress
assertServiceBindingOrphanMitigationSuccess
assertServiceBindingOrphanMitigationFailure
```

Reply:
Done.